### DISCUSSION TITLE
Custom Persian Fonts for READMEs: A Request

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#65430] [Other Feature Feedback, Questions, & Ideas]
Hello, I wanted to request the ability to use a Persian custom font for README files viewed on Github. As an avid Github user who primarily writes documentation and READMEs in Persian, it would be extremely helpful to have a font option better suited for the Persian script. The default fonts on Github are very well designed, but not optimized for the cursive style of Persian. Having a font with clearer joins and ligatures between letters would greatly improve the reading experience. I wanted to suggest adding the Persian font "Vazir" as a README custom font option. Vazir is an open source Persian font that maintains readability at small sizes and resolves many of the legibility issues encountered with other fonts. Given the significant number of Persian developers and open source projects on Github, adding Vazir or a similar customized Persian font would make Github much more inclusive for this community. The ideal implementation could be allowing a per-repo Persian README font setting. This way, repositories with primarily Persian documentation can enable the customized font, while retaining the standard font for other languages. Please consider adding a Persian README font option like Vazir. 

This link is for the "Vazir" font on Google Fonts.
https://fonts.google.com/specimen/Vazirmatn

Thank you for your time.

**Comment by aliprogammerali** [comment]
I also need this feature.

**Comment by MohsenDastaran** [comment]
Great Idea.

**Comment by HamiDev** [comment]
Good idea. "Vazir" persian font is the best Farsi font.
[Vazir font on github](https://rastikerdar.github.io/vazirmatn/)

**Comment by peymanath** [comment]
I also need this feature. 

[Vazirmatn font Repository](https://rastikerdar.github.io/vazirmatn/en)

**Comment by HosseinJavooni** [comment]
absolutely essential!

**Comment by ariana010111** [comment]
Great Idea. I need too

**Comment by Pourghorban** [comment]
I also need this feature.

**Comment by Pourghorban** [comment]
I also need this feature.

**Comment by ghost** [comment]
I also need it👌🏼

**Comment by nafasebra** [comment]
I also need this feature.

**Comment by aliprogammerali** [comment]
Up

**Comment by mhdisharifi** [comment]
this feature is good. i also need it

**Comment by meiadnoushadi** [comment]
🙏🏻

**Comment by aliprogammerali** [comment]
Dear GitHub Team, Please review this request.
@Iron-Ham @HenSquared @jlord @cfahlbusch @iskae @broccolini @howdyray @zrdaley
